app.go

```diff
@@ -13,7 +13,11 @@ import "time"
 
 // SetName for the server
 func (app *App) SetName(name string) {
-	app.name = name
+	if len(name) > 0 {
```

Comment:
this check breaks ability to set `name` to `""`

Reply:
Yep! But logic in old (removed) code also silently discards empty app.name :)

Reply:
hehe, yes, my bad :) but I think if we here, we should fix it too :slightly_smiling_face:

Reply:
`OptAppName` option added
`App.SetName()` fixed and marked as deprecated